plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,144 @@
+# Analytics Module: Google Analytics
+
+This plugin provides an opinionated implementation of the Backstage Analytics
+API for Google Analytics. Once installed and configured, analytics events will
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage app:
+   `cd packages/app && yarn add @backstage/plugin-analytics-module-ga`
+2. Add the API implementation to your App and configure the plugin (see below).
```

Comment:
Dividing it into three steps can make the getting started more logical (fewer back and forth between code and configuration). Do you agree?
1. Installation
2. Configuration
3. Usage example

Reply:
I think I get what you're going for.  I updated things a bit accordingly.